src/ffi/logging/test.rs

```diff
@@ -0,0 +1,42 @@
+/// Added in external crate to avoid line changes as much as possible
+use tracing::{debug, trace};
+
+use super::*;
+
+use std::{
+    fmt::Debug,
+    sync::{Arc, Mutex},
+};
+
+use crate::{TelioLogLevel, TelioLoggerCb};
+
+#[test]
+fn test_trace_via_telio_cb() {
+    let expected = [(
```

Comment:
It doesn't seem like this is used

Reply:
Ah missed assert somehow

Reply:
Forgot to commit

Reply:
yeh forgot to remove, though originaly that it would be simpler to maintain, but also so  did not see tests/loger.rs,
but i prefer to have tests next to code